thi-app/components/Sidebar.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    Image
+} from 'react-native';
+
+export default function Sidebar() {
+    return(
+        <View className="flex-1 justify-center bg-[#17468F] w-[339px] h-[834px]">
```

Comment:
Avoid using pixels for dimensions for containers if you can, use percentage and breakpoint prefix to dynamically resize your page to different orientations/screens (table vs phone). Refer to this: https://tailwindcss.com/docs/responsive-design